cli/runner/cmd/requests.go

```diff
@@ -117,3 +143,14 @@ func panicCheck(err error) {
 		panic(err)
 	}
 }
+
+func extractExtension(filename string) string {
+	var ret string
+	f := []rune(filename)
+	for i := len(f) - 1; i >= 0; i -= 1 {
+		if f[i] == '.' {
+			return string(f[i+1:])
+		}
+	}
```

Comment:
You could use [`strings.Split(filename, ".")`](https://pkg.go.dev/strings#Split) for this.

Reply:
My reasoning behind doing it this way was in case the filename was some sort of path such as '../../file.py' or something, but if strings.Split(...) is still preferable I'd be happy to change it

Reply:
Ah I see. I wasn't thinking of that case, good point. I still think if you just grab the last element split (i.e. `arr[len(arr)-1]` it would handle that too?
This is what I mean: https://go.dev/play/p/HK9Pz_nQlTK

But LMK if I'm maybe missing something.

Reply:
Oh I see it now, that makes a lot of sense. That method is probably a lot easier to read in code as well. The only thing that I'm thinking is that the way it's implemented now might be more memory efficient (but I could be wrong, I'm still not super familiar with slices) because it doesn't have to create a new array. The current implementation might also be a bit faster because it just checks for the first '.' rather than all of them. 

I do understand, however, that clean and readable code can far outweigh any of these pros, so I'll defer to your judgement on this one

Reply:
Good point, that convinces me. No need to change this in that case!